src/config/nunjucks.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     const nunjucksEnv = nunjucks.configure(viewsPath, {
       autoescape: true,
       express: app,
-      noCache: true,
+      noCache: process.env.NODE_ENV !== "production",
```

Comment:
Makes sense - it doesn't look like this was a calculated decision to have enabled in prod, as far as I can tell.

One note of caution - by tying to `NODE_ENV`, it's not possible to override this on a per-environment basis. For example, it'll be enabled in all environments together. In general it's nicer to have explicit config for this sort of thing - `TEMPLATE_CACHING` or something.